components/sections/Contact.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 
 const ContactSection = () => {
   return (
-    <section className='mb-10'>
+    <section className='mb-10' data-cy='contact'>
```

Comment:
It should be consistent follow either `className then data` / `data then className`

Reply:
data then className